modules/settings/assets/css/style.css

```diff
@@ -1,18 +1,24 @@
 body {
 	background: #fff;
 }
+
+#wpfooter {
+	display: none;
```

Comment:
Looking forward we'll probably add a feedback link there like we have in IO. Already discussed it somewhere in Slack, so why?